src/DashboardUI/src/pages/application/estimation-tool/EstimationToolNavbar.tsx

```diff
@@ -0,0 +1,36 @@
+import { mdiArrowLeft, mdiWater } from '@mdi/js';
+import Icon from '@mdi/react';
+import { Button, Container, Nav, Navbar } from 'react-bootstrap';
+
+interface EstimationToolNavbarProps {
+  navigateToWaterRightLandingPage: () => void;
+}
+
+export function EstimationToolNavbar(props: EstimationToolNavbarProps) {
+  return (
+    <Navbar className="p-0 second-nav estimate-tool-navbar">
+      <Container fluid>
+        <Nav>
+          <Nav.Item onClick={props.navigateToWaterRightLandingPage}>
+            {/* render as <button> instead of <a role="button"> */}
+            <Nav.Link as="button" className="text-dark">
+              <div className="d-flex flex-row align-items-center gap-2">
```

Comment:
Default is row for flex direction so that class can be removed if you'd like
```suggestion
              <div className="d-flex align-items-center gap-2">
```

Reply:
Same goes for a few others I see below

Reply:
that's fine, i'll remove it